docs/static/monitoring/monitoring-internal-legacy.asciidoc

```diff
@@ -0,0 +1,152 @@
+[role="xpack"]
+[[monitoring-internal-collection-legacy]]
+=== Collect {ls} monitoring data using legacy internal collectors
+++++
+<titleabbrev>Legacy internal collection</titleabbrev>
+++++
+
+IMPORTANT: Use <<monitoring-internal-collection,internal collection>> or
+<<monitoring-with-metricbeat, {metricbeat} collection>> instead of legacy
+internal collection. If you are currently using legacy internal collection, you
+should migrate to either Metricbeat collection or internal collection.
+
+==== Migrating from legacy internal collection to new internal collection
+
+Migrating from legacy internal collection to new
+<<monitoring-internal-collection,Internal collection>> is straightforward.  Drop
+the `xpack` prefix from your configuration settings. 
+For example, `xpack.monitoring.enabled` is now `monitoring.enabled`.
+See <<monitoring-settings, Monitoring Settings>> for the list of settings.
+
+If you don’t have an Elasticsearch output plugin configured in the pipelines,
+add the setting `monitoring.cluster_uuid` to your logstash.yml.
```

Comment:
Do we need to give an explanation of what this should be, and how they can retrieve it?

Reply:
Not sure how much we should invest in the legacy component as it is no longer one of the preferred approaches. Let's discuss.

Reply:
True, although we could just use the description from the newer mechanism